src/execute_linter.cc

```diff
@@ -24,24 +25,172 @@
 #include "absl/strings/string_view.h"
 #include "src/lint_errors.h"
 #include "src/linter.h"
+#include "src/linter_options.h"
 #include "zetasql/base/status.h"
 #include "zetasql/base/status_macros.h"
 
 namespace zetasql::linter {
 
 namespace {
+
+// This function will parse "NOLINT"(<CheckName>) syntax and adjust enability
+// options for each check. If NOLINT usage errors occur, it counts as a lint
+// error and it will be returned in a result.
+// This works similar to consistent comment type check.
+LinterResult ParseNoLintComments(absl::string_view sql, LinterOptions options,
+                                 std::map<ErrorCode, LinterOptions>* nolint) {
```

Comment:
Use absl containers https://abseil.io/docs/cpp/guides/container#abseil-containers
I guess, you don't really need entries sorted, so then absl::flat_hash_map is your choice.

Reply:
Sure. That would make it faster.